python/cusignal/signaltools.py

```diff
@@ -1226,7 +1226,7 @@ def cmplx_sort(p):
     return take(p, indx, 0), indx
 
 
-def resample(x, num, t=None, axis=0, window=None):
+def resample(x, num, t=None, axis=0, window=None, fft=False):
```

Comment:
Let's make `fft` a clearer variable. I'd rather this be `domain` with options `time` and `frequency`. Let's consider changing this to:

```python
def resample(x, num, t=None, axis=0, window=None, domain='time')
```